tests/components/UniversalResults.test.tsx

```diff
@@ -33,16 +33,16 @@ describe('UniversalResults', () => {
 
   it('Results are displayed', () => {
     render(<UniversalResults verticalConfigMap={{}} />);
-    const verticals = mockedState.universal.verticals;
+    const verticals = mockedState.universal?.verticals ?? [];
     expect(screen.getByText(verticals[0].verticalKey)).toBeDefined();
     expect(screen.getByText(verticals[1].verticalKey)).toBeDefined();
     expect(screen.queryByText(verticals[2].verticalKey)).toBeNull();
 
     expect(screen.queryByText(verticals[0].appliedQueryFilters[0].displayValue)).toBeNull();
 
     function checkResultData(resultData: Record<string, unknown>) {
-      expect(screen.getByText(resultData.name.toString())).toBeDefined();
-      expect(screen.getByText(resultData.description.toString())).toBeDefined();
+      expect(typeof resultData.name === 'string' && screen.getByText(resultData.name)).toBeDefined();
```

Comment:
I'm a little wary of this format of `expect(blah && blah).toBeDefined();` since `expect(false).toBeDefined()` would still pass even though we intend to assess the actual return value of the second part of the condition. Is there something else we can do? like changing the type or the way we assert things here

Reply:
changed it to assert `toBeTruthy` instead of `toBeDefined` so the false case won't pass